### PR TITLE
Feature/get user

### PR DESCRIPTION
**Pull request by Bandisa**
# Task 
implemented the end point for get user
 this will fetch a user from db given a request with userId as per interface
# Testing
> on root, run:
```sh
 yarn test libs/api/core/feature/test/users.functions.test.ts
```

### CODE REVIEW

File: libs/api/core/feature/test/memories.functions.test.ts
Comment (ArmandKrynauw) [inline libs/api/core/feature/test/memories.functions.test.ts:1]: I will merge these tests but it being hardcoded is technically incorrect to me.

File: libs/api/core/feature/test/users.functions.test.ts
Comment (ArmandKrynauw) [inline libs/api/core/feature/test/users.functions.test.ts:1]: Same objection as the other tests

File: libs/api/memories/feature/src/commands/create-memory.handler.ts
Comment (ArmandKrynauw) [inline libs/api/memories/feature/src/commands/create-memory.handler.ts:1]: You seem to be writing the data for the new memory to the database in the command handler. The data flow should be that and event is emitted and then that event will save the data to the database. You also call create and commit on the event publisher but I can't see an event being applied/emitted anywhere by the profile model. I would suggest looking at the data/event flow of the create profile command.

File: libs/api/users/feature/src/commands/get-user.handler.ts
Comment (ArmandKrynauw) [inline libs/api/users/feature/src/commands/get-user.handler.ts:1]: The getUser endpoint should be a query and not a command. It should go through the CQRS query handler. You also seem to call model.create which will emit a new user created event. Look at how queries are implemented in the profiles, specifically the getUserProfile query.

File: seed.js
Comment (ArmandKrynauw) [inline seed.js:1]: This kinda needs to be remove and not committed to the repo.

**Review by ArmandKrynauw — CHANGES_REQUESTED** [review]
Check some of the comments added

File: libs/api/core/feature/test/memories.functions.test.ts
Comment (Bandisa) [reply libs/api/core/feature/test/memories.functions.test.ts:1]: Noted.

File: seed.js
Comment (Bandisa) [reply seed.js:1]: will add it to .gitignore

File: libs/api/users/feature/src/commands/get-user.handler.ts
Comment (Bandisa) [reply libs/api/users/feature/src/commands/get-user.handler.ts:1]: will implement it as a query.

File: libs/api/memories/feature/src/commands/create-memory.handler.ts
Comment (Bandisa) [reply libs/api/memories/feature/src/commands/create-memory.handler.ts:1]: will do.

**Pull request closed without merging**